plugins/plugin-chart-echarts/src/Pie/transformProps.ts

```diff
@@ -140,11 +140,14 @@ export default function transformProps(chartProps: EchartsPieChartProps): PieCha
       timeFormatter: getTimeFormatter(dateFormat),
     });
 
+    const isFiltered = filterState.selectedValues && !filterState.selectedValues.includes(name);
+
     return {
       value: datum[metricLabel],
       name,
       itemStyle: {
         color: colorFn(name),
+        opacity: isFiltered ? 0.3 : 1,
```

Comment:
the `opacity` should be added to echart constants.

Reply:
In https://github.com/apache-superset/superset-ui/pull/1205, a constant about opacity was extracted.  I think we can wait for this PR merge to modify this PR.